securedrop_client/gui/widgets.py

```diff
@@ -2034,11 +2034,14 @@ def _on_update_authenticated_user(self, user: User) -> None:
         """
         When the user logs in or updates user info, update the reply badge.
         """
-        if user.uuid == self.sender.uuid:
-            self.sender_is_current_user = True
-            self.sender = user
-            self.sender_icon.setToolTip(self.sender.fullname)
-            self._update_styles()
+        try:
+            if self.sender and self.sender.uuid == user.uuid:
+                self.sender_is_current_user = True
+                self.sender = user
+                self.sender_icon.setToolTip(self.sender.fullname)
+                self._update_styles()
+        except sqlalchemy.orm.exc.ObjectDeletedError:
```

Comment:
This probably isn't necessary anymore. I'll look into removing this Monday after more testing. It doesn't hurt but it's adding code complexity that isn't needed if we ensure that a Reply must always have an associated Sender account.